backbone.js

```diff
@@ -644,7 +646,7 @@
       if (attrs && wait) this.attributes = _.extend({}, attributes, attrs);
 
       var method = this.isNew() ? 'create' : (options.patch ? 'patch' : 'update');
-      if (method === 'patch' && !options.attrs) options.attrs = attrs;
+      if (method === 'patch' && !options.attrs) options.attrs = this.changedAttributes(attrs);
```

Comment:
Thanks, I fixed it.
 But,  i want the method work like that, and overrided model's set function in my app. Maybe not the best way : )